### PR TITLE
Add support for mh lb_algo

### PR DESCRIPTION
**Pull request by MrFreezeex**
Signed-off-by: Arthur Outhenin-Chalandre <arthur.outhenin-chalandre@cern.ch>

#### Pull Request (PR) description

Add support for mh lb_algo, also add related mh_port and mh_fallback similarly to sh_port and sh_fallback.

### CODE REVIEW

File: manifests/lvs/virtual_server.pp [hunk 1]
```diff
@@ -25,6 +25,10 @@
 #
 # @param omega
 #
+# @param mh_port
+#
```
Comment (smortex) [inline manifests/lvs/virtual_server.pp:29]:
```suggestion
#   Enable mh-port for mh scheduler
```

File: manifests/lvs/virtual_server.pp [hunk 2]
```diff
@@ -25,6 +25,10 @@
...
+# @param mh_fallback
+#
```
Comment (smortex) [inline manifests/lvs/virtual_server.pp:31]:
```suggestion
#   Enable mh-fallback for mh scheduler
```

**Review by smortex — APPROVED** [review]
Looks good as far as someone who has zero experience with keepalived is concerned.

The module lacks a lot of documentation, I suggest adding some for these new parameters.  I copied these from the man page available at https://man.archlinux.org/man/keepalived.conf.5.en but fill free to rephrase for better description :wink:

**Comment by MrFreezeex** [comment]
> Looks good as far as someone who has zero experience with keepalived is concerned.
> 
> The module lacks a lot of documentation, I suggest adding some for these new parameters. I copied these from the man page available at https://man.archlinux.org/man/keepalived.conf.5.en but fill free to rephrase for better description wink

Thanks for the suggestion I also added those for the sh options in a separated commit!

**Pull request merged**